frontend/simple/js/state.js

```diff
@@ -189,66 +221,74 @@ const actions = {
     {dispatch, commit, state}: {dispatch: Function, commit: Function, state: Object},
     {contractId, hash, entry}: {contractId: string, hash: string, entry: Object}
   ) {
-    // verify we're expecting to hear from this contract
-    if (!state.pending.includes(contractId) && !state.contracts[contractId]) {
-      // TODO: use a global notification system to both display a notification
-      //       and throw an exception and write a log message.
-      return console.error(`NOT EXPECTING EVENT!`, contractId, entry)
-    }
-    const type = entry.type
-    entry = (Events[type] || contracts[type]).fromObject(entry, hash)
-
-    if (Events.HashableContract.prototype.isPrototypeOf(entry)) {
-      console.log(`handleEvent for new ${type}:`, entry)
-      if (entry.toObject().parentHash) {
-        // TODO: use a global notification object to handle this and all other errors,
-        //       and figure out if an exception should be thrown
-        return console.error(`${type} has non-null parentHash!`, entry)
+    if (loggingOut) return
```

Comment:
We just drop events if the user is logging out? That's not OK. Also we need to remove this variable anyway